atom/browser/api/atom_api_top_level_window.cc

```diff
@@ -1174,7 +1184,8 @@ void TopLevelWindow::BuildPrototype(v8::Isolate* isolate,
       .SetMethod("setThumbnailToolTip", &TopLevelWindow::SetThumbnailToolTip)
       .SetMethod("setAppDetails", &TopLevelWindow::SetAppDetails)
 #endif
-      .SetProperty("id", &TopLevelWindow::GetID);
+      .SetProperty("id", &TopLevelWindow::GetID)
+      .SetProperty("tabbingIdentifier", &TopLevelWindow::GetTabbingIdentifier);
```

Comment:
i think there's a mismatch here in platform defines: it's guarded in the header but not in the impl. I think i'd just recommend exposing it for mac-only as opposed exposing it across platforms and having it no-op where not applicable